yt/yt/client/api/security_client.h

```diff
@@ -76,7 +76,9 @@ struct TSetUserPasswordOptions
 
 struct TIssueTokenOptions
     : public TTimeoutOptions
-{ };
+{
+    std::optional<TString> Description;
```

Comment:
I don't have a strong opinion here, but empty string with empty description semantics seems natural to me, so std::optional can be removed.